puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -23,29 +28,18 @@
     notify  => Exec["jenkins-jobs-update-${config_name}"],
   }
 
-  cron { "jenkins-jobs-update-${config_name}-delete-old":
-    ensure      => absent,
```

Comment:
I thought maybe we could miss some cleanup by removing the crons with `ensure => absent` but after checking https://github.com/theforeman/foreman-infra/commit/7495ec8373e6606dfbce94b939fbd215644b9bc9 I understand the logic here